jwst/datamodels/schemas/psubarray.schema.yaml

```diff
@@ -0,0 +1,22 @@
+type: object
```

Comment:
Yes, it should.  I had it originally as the whole section but realized we only need the name of the subarray and not the other attributes.